File: cassdio-web/src/main/java/kr/hakdang/cassdio/web/CassdioWebApplication.java
package kr.hakdang.cassdio.web;

import kr.hakdang.cassdio.common.PackageConstants;
import kr.hakdang.cassdio.web.config.CassandraDefaultConfigurationProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@EnableConfigurationProperties(CassandraDefaultConfigurationProperties.class)
@ConfigurationPropertiesScan(basePackages = PackageConstants.BASE_PACKAGE)
@SpringBootApplication(scanBasePackages = {PackageConstants.BASE_PACKAGE})
public class CassdioWebApplication {

    public static void main(String[] args) {
        SpringApplication.run(CassdioWebApplication.class, args);
    }

}

File: cassdio-web/src/main/kotlin/kr/hakdang/cassdio/web/CassdioWebApplication.kt
package kr.hakdang.cassdio.web

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.cassandra.CassandraAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(
    scanBasePackages = ["kr.hakdang.cassdio"],
    exclude = [CassandraAutoConfiguration::class],
)
class CassdioWebApplication

fun main(args: Array<String>) {
    runApplication<CassdioWebApplication>(*args)
}
